pkgs/development/compilers/llvm/11/default.nix

```
@@ -125,7 +125,7 @@ let
       cc = tools.clang-unwrapped;
       libcxx = targetLlvmLibraries.libcxx;
       extraPackages = [
-        targetLlvmLibraries.libcxxabi
+        libcxx.cxxabiLib
```

I think you want
```suggestion
        targetLlvmLibraries.libcxx.cxxabiLib
```
?

---

They're the same thing because of the `rec`, but I'll happily change it to be more explicit if you think it'd be better.